install.ps1

```diff
@@ -81,7 +81,33 @@ switch ($architecture) {
 
 $pkgInfo = Invoke-WebRequest "https://registry.npmjs.org/@pnpm/exe" -UseBasicParsing
 $versionJson = $pkgInfo.Content | ConvertFrom-Json
-$version = $versionJson.'dist-tags'.latest
+$versions = Get-Member -InputObject $versionJson.versions -Type NoteProperty | Select-Object -ExpandProperty Name
+$distTags = Get-Member -InputObject $versionJson.'dist-tags' -Type NoteProperty | Select-Object -ExpandProperty Name
+
+$version = $null
+$preferredVersion = "latest"
+
+if ($null -ne $env:PNPM_VERSION -and $env:PNPM_VERSION -ne "") {
+  $preferredVersion = $env:PNPM_VERSION
+}
+
+if ($null -eq $version -and $preferredVersion -in $distTags) {
+  $version = $versionJson.'dist-tags' | Select-Object -ExpandProperty $preferredVersion
+}
+
+if ($null -eq $version -and $preferredVersion -in $versions) {
```

Comment:
shouldn't this be an `else if`?

Reply:
PowerShell does not have "normal" ```else if```. I would have to nest ```if``` into ```else``` block. In this case ```$null -eq $version``` check is sufficient

Reply:
PowerShell does not have "normal" else if. I would have to nest if into else block. In this case $null -eq $version check is sufficient